### PR TITLE
[Feature] Add merge_heatmap_to_5

### PR DESCRIPTION
**Pull request by Yshuo-Li**
For DIC

### CODE REVIEW

File: mmedit/models/extractors/feedback_hour_glass.py [hunk 1]
```diff
@@ -88,3 +89,55 @@ def forward(self, x):
...
+        nose
+        mouse
+        face silhouette
+
+    Args:
+        ori_heatmap (Tensor): Input heatmap tensor. B*N*32*32.
+        detach (bool): Detached from the current tensor or not.
+
+    returns:
+        Tensor: New heatmap.
```
[the remark this answers is not in the document]
Comment (Yshuo-Li) [reply mmedit/models/extractors/feedback_hour_glass.py:111]: yes

File: mmedit/models/extractors/feedback_hour_glass.py [hunk 2]
```diff
@@ -88,3 +89,55 @@ def forward(self, x):
...
+    """
+    # landmark[36:42], landmark[42:48], landmark[27:36], landmark[48:68]
+    heatmap = ori_heatmap.clone()
+    max_heat = heatmap.max(dim=2, keepdim=True)[0].max(dim=3, keepdim=True)[0]
+    max_heat = torch.max(max_heat, torch.ones_like(max_heat) * 0.05)
```
[the remark this answers is not in the document]
Comment (Yshuo-Li) [reply mmedit/models/extractors/feedback_hour_glass.py:116]: how about `max_heat = max_heat.clamp_min_(0.05)`

File: mmedit/models/extractors/feedback_hour_glass.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (ckkelvinchan) [reply mmedit/models/extractors/feedback_hour_glass.py:111]: Maybe we can add this in docstring.

File: mmedit/models/extractors/feedback_hour_glass.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (ckkelvinchan) [reply mmedit/models/extractors/feedback_hour_glass.py:116]: Oh I mean whether this 0.05 is a standard value or referenced from somewhere? And `max_heat = max_heat.clamp_min_(0.05)` sounds good.

File: mmedit/models/extractors/feedback_hour_glass.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (Yshuo-Li) [reply mmedit/models/extractors/feedback_hour_glass.py:111]: done

**Review by ckkelvinchan — APPROVED** [review]

**Comment by innerlee** [comment]
Need rebase

**Pull request merged**